indico/web/forms/validators.py

```diff
@@ -379,6 +379,14 @@ class SoftLength(Length):
     where surrounding whitespace is stripped before validation.
     """
 
+    def __call__(self, form, field):
+        _data = field.data
```

Comment:
I don't really like `_foo` names inside local scopes ;)

What's wrong with `orig_data` which is clearer about its purpose IMO?

Reply:
👍